woocommerce/woocommerce-android
# Add toolbar for AddOrderNoteFragment

Part of #10570 

### Description

This PR introduces individual toolbars for AddOrderNoteFragment screen, replacing the shared toolbar previously used with MainActivity. This modification is essential to accommodate screens within a two-pane layout, ensuring that each screen maintains its distinct toolbar despite sharing a common parent.

### Testing instructions

Navigate to the order note editing (Order list -> Order detail -> Add a note) screen and ensure the toolbar and its action work as expected on both phone and tablet.

### Images/gif
